.circleci/config.yml

```diff
@@ -605,35 +598,53 @@ jobs:
             - *restore_cache
             - *add_ssh_keys
             - *prepare_environment
+            - run:
+                name: Collect Instance Role And check If Instance Was Created
+                when: always
+                command: |
+                  echo 'export INSTANCE_ROLE="Demisto GA"' >> $BASH_ENV
+                  export TEMP=$(cat ./Tests/filter_envs.json | jq '."Demisto GA"')
+                  echo "Demisto GA filter=$TEMP"
+                  if [[ "$TEMP" == "true" ]];
+                    then
+                      echo 'export INSTANCE_WAS_CREATED="True"' >> $BASH_ENV
+                      exit 0
+                  fi
+                  source $BASH_ENV
```

Comment:
Why do we not put this as part of the configure step? why does it need a whole step?

Reply:
1. Because these env variables are different from server to server.
2. Because these env variables are needed for the waiting step, configuration step, testing step and the destroy instance step.